opensrp-tasking/src/test/java/org/smartregister/tasking/presenter/TaskingHomePresenterTest.java

```diff
@@ -0,0 +1,945 @@
+//        assertEquals("2019-07-04", actualCardDetails.getStartDate());
+//        assertEquals("2019-08-05", actualCardDetails.getEndDate());
+//        assertEquals(MOSQUITO_COLLECTION, actualCardDetails.getInterventionType());
+//    }
+//
+//    @Test
+//    public void testOnIRSVerificationCardDetailsFetched() {
+//        IRSVerificationCardDetails expectedCardDetails = new IRSVerificationCardDetails(NOT_VISITED,
+//                "yes", "no", "sprayed", "No chalk",
+//                "No sticker", "No card");
+//
+//
+//        presenter.onCardDetailsFetched(expectedCardDetails);
+//
+//        verify(view).openCardView(cardDetailsArgumentCaptor.capture());
+//
+//        IRSVerificationCardDetails actualCardDetails = (IRSVerificationCardDetails) cardDetailsArgumentCaptor.getValue();
+//
+//        assertEquals(NOT_VISITED, actualCardDetails.getStatus());
+//        assertEquals("yes", actualCardDetails.getTrueStructure());
+//        assertEquals("no", actualCardDetails.getEligStruc());
+//        assertEquals("sprayed", actualCardDetails.getReportedSprayStatus());
+//        assertEquals("No chalk", actualCardDetails.getChalkSprayStatus());
+//        assertEquals("No sticker", actualCardDetails.getStickerSprayStatus());
+//        assertEquals("No card", actualCardDetails.getCardSprayStatus());
+//
+//    }
+
+    @Test
+    public void testOnLocationValidatedForMarkStructureIneligible() {
+
+        Whitebox.setInternalState(presenter, "markStructureIneligibleConfirmed", true);
+        assertTrue(Whitebox.getInternalState(presenter, "markStructureIneligibleConfirmed"));
+        presenter = spy(presenter);
+        presenter.onLocationValidated();
+
+        verify(presenter).onMarkStructureIneligibleConfirmed();
+        assertFalse(Whitebox.getInternalState(presenter, "markStructureIneligibleConfirmed"));
+    }
+
+    @Test
+    public void testOnLocationValidatedForRegisterFamily() {
+
+        Whitebox.setInternalState(presenter, "selectedFeatureInterventionType", REGISTER_FAMILY);
+
+        presenter.onLocationValidated();
+
+        verify(view).registerFamily();
+    }
+
+    @Test
+    public void testOnLocationValidatedForCardDetailsWithChangeInterventionStatusFalse() {
+
+        Whitebox.setInternalState(presenter, "selectedFeatureInterventionType", PAOT);
+        Whitebox.setInternalState(presenter, "changeInterventionStatus", false);
+        presenter = spy(presenter);
+        presenter.onLocationValidated();
+
+        verify(presenter).startForm(featureArgumentCaptor.capture(), cardDetailsArgumentCaptor.capture(), stringArgumentCaptor.capture());
+        assertNull(cardDetailsArgumentCaptor.getValue());
+        assertEquals(PAOT, stringArgumentCaptor.getValue());
+    }
+
+//    @Test
+//    public void testOnlocationValidatedForCardDetailWithChangeInterventionTrue() {
+//        FamilyCardDetails expectedCardDetails = new FamilyCardDetails(COMPLETE, "19 Jan 1970", "nifi-user");
```

Comment:
I am not sure of the intention when commenting these out. Could you do something about it? Create an issue, todo, remove, fix